packages/components/src/components/Alert/Alert.tsx

```diff
@@ -229,31 +252,32 @@ export const Alert: FC<AlertProps> = ({
       <View style={{ flexDirection: 'row' }}>
         {iconDisplay}
```

Comment:
Per the item I added to [the Q&A thread](https://adhoc.slack.com/archives/C05DQPHUVNZ/p1709747499865419), iconDisplay via either the icon itself or iconProps should have preventScaling true.

Reply:
Added preventScaling to iconDisplay above.